pages/portfolio.html

```diff
@@ -1,12 +1,37 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>my cool portfolio</title>
+    <!-- Link to Style Sheet -->
+    <link href="/styles/main-style.css" rel="stylesheet" />
+    <link href="/styles/portfolio-style.css" rel="stylesheet" />
 </head>
-<body>
-  
+<body>  
+  <header>
+    <nav>
+
+      <a href="about.html">about me</a> 
+      <a href="portfolio.html">portfolio</a> 
+      <a href="index.html">home</a>
+      
+    </nav>      <h1>my workio</h1>
+  </header>
+  <main>
+    <div id="container">
+
+      
+      <div class="project"><h1>project</h1></div>
+      <div class="project"><h1>project</h1></div>
+      <div class="project"><h1>project</h1></div>
+
+
+    </div>
```

Comment:
Let's make the `div.container` a `section` instead

Also there should only ever be one `h1` per page, so perhaps `h2` would be better here